### ISSUE TITLE
Easier translation through Crowdin

### ISSUE THREAD

**Issue by bubblineyuri** [glitch-soc/mastodon#1313]
It would be very nice if we could use Crowdin (which Mastodon uses) to translate glitch-soc. I would then do some parts for the German translation (for strings that aren't available in default Mastodon), because I am slowly beginning to switch over my entire infrastructure to glitch-soc and we have some German-speaking users there as well.

**Comment by ThibG** [comment]
Yeah, this would be a good thing, but unfortunately, the whole translation situation for glitch-soc is a mess, because:
- on the back-end side, we share the same file as upstream does, so if we did plug into Crowdin or something, we'll display not only glitch-soc-specific strings, but upstream strings as well, which is likely to be a nightmare when merging changes
- on the JS side, we use separate files which include upstream's. While this works fine for *use* and seems cleaner in theory, the tooling doesn't handle that *at all*, so for now we're stuck with manual translation

I'm not sure how to improve the situation, tbh :/

**Comment by comradekingu** [comment]
Please use Weblate instead, it is a much better tool, and it is libre software that doesn't spy on the user. Mastodon Florence, Mastodon-Twitter Crossposter and Invidious are already on hosted.weblate.org/ :)

**Comment by bubblineyuri** [comment]
I just don't like how Weblate looks and it doesn't have this nifty one-click translation feature that saved me a lot of time figuring out what Eugen meant

**Comment by comradekingu** [comment]
@bubblineyuri Weblate has plenty of editors, and recently overhauled its look.
The Zen editors are a lot faster than what Crowdin offers with its paged UI. I don't really know what you mean by one-click?
